cmd/loadtest/loadtest.go

```diff
@@ -1559,7 +1559,11 @@ func configureTransactOpts(tops *bind.TransactOpts) *bind.TransactOpts {
 		if ltp.ForceGasPrice != nil && *ltp.ForceGasPrice != 0 {
 			tops.GasPrice = big.NewInt(0).SetUint64(*ltp.ForceGasPrice)
 		} else {
-			tops.GasPrice = big.NewInt(0).Add(ltp.BaseFee, ltp.CurrentGasTipCap)
+			if ltp.BaseFee != nil {
+				tops.GasPrice = big.NewInt(0).Add(ltp.BaseFee, ltp.CurrentGasTipCap)
+			} else {
+				log.Fatal().Msg("EIP-1559 not activated. Please use --legacy")
```

Comment:
@leovct noticed the command was hanging (actually panic-ing in trace mode) when we send _non-legacy_ txs to _non EIP-1559_ enabled chains. this error is the same behavior as `cast`